databricks/koalas/config.py

```diff
@@ -134,10 +134,12 @@ def validate(self, v: Any) -> None:
     Option(
         key="compute.max_rows",
         doc=(
-            "'compute.max_rows' sets the limit of the current DataFrame. Set `None` to unlimit "
-            "the input length. When the limit is set, it is executed by the shortcut by "
-            "collecting the data into driver side, and then using pandas API. If the limit is "
-            "unset, the operation is executed by PySpark. Default is 1000."
+            "'compute.max_rows' sets the limit of the current Koalas DataFrame. Set `None` to "
+            "unlimit the input length. When the limit is set, it is executed by the shortcut by "
+            "collecting the data into the driver, and then using the pandas API. If the limit is "
+            "unset, the operation is executed by PySpark. Note that this option does not impact "
+            "the sizes of the Pandas DataFrames/Series received by functions of GroupBy. "
```

Comment:
I suggested to remove this because other friends like pandas DataFrame/Series via `DataFrame.apply_batch` also don't get affected by this configuration. Elaborating these cases all looks slightly an overkill, and I think "Koalas DataFrame on driver side" clarifies it enough.